examples/isomorphic-flux-chat/js/main.js

```diff
@@ -0,0 +1,25 @@
+var mockData = require('./mock-data')
+var Nuclear = require('nuclear-js')
+var reactor = new Nuclear.Reactor({
+  debug: true,
+})
+window.reactor = reactor
+var Chat = require('./modules/chat')
+
+var ChatApp = require('./components/ChatApp.react')
+ChatApp = Nuclear.provideReactor(ChatApp)
```

Comment:
Ah, thanks so much! Guess I'm quite behind the times here =) so much happens in a month or two... 

Ah, so the child has to use nuclearMixin or nuclearComponent because they both include `contextType` objs?